src/plugins/contributorsPlugin/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "gatsby-starter-plugin",
```

Comment:
This isn't accurate, is it?

Reply:
Thank you for pointing this out, the name has been fixed to "github-contributors-plugin".